vars/main.yml

```diff
@@ -385,7 +385,7 @@ basebackup:
   - { option: "max-rate", value: "100M" }
   - { option: "checkpoint", value: "fast" }
 pg_probackup:
```

Comment:
Hello @vitabaks,

I have observed that the original commands `pg_probackup[0].value` and `pg_probackup_patroni_cluster_bootstrap_command` are identical.

Am I correct in understanding this?

I have refactor the variables, but now they both inherit from the same base `pg_probackup_command_parts`.

If you could kindly confirm that this is satisfactory, I would appreciate it. I have added tests to ensure the expected outcome, and they match. Therefore, the final results are the same as before my refactoring.

Reply:
yes, it's the same restore command